kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/food2door/FoodOrder.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.good.patterns.challenges.food2door;
+
+import java.util.Map;
+
+public class FoodOrder {
+
+    private Map<Product, Integer> orderContents;
+    private Integer orderNo;
+
+    public FoodOrder(Map<Product, Integer> orderContents, Integer orderNo) {
+        this.orderContents = orderContents;
+        this.orderNo = orderNo;
+    }
+
+    public double foodOrderPrice() {
+        double price = 0;
+        for (Map.Entry<Product, Integer> entry : orderContents.entrySet()) {
+            price += entry.getKey().getPrice() * entry.getValue();
+        }
+        return (double) Math.round(price * 100) / 100;
```

Comment:
jeśli chodzi o zaokrąglenie do 2 miejsc po przecinku to:
1) Math.round() 
2) BigDecimal jeśli mamy do czynienia z pieniędzmi. Zauważ, że z tym zaokrągleniem możesz coś tracić. Na przykład w Hiszpanii paliwo jest sprzedawane z pracyzją do 3 miejsc po przecinku.